qa/suites/fs/upgrade/upgraded_client/README

```diff
@@ -0,0 +1,9 @@
+This suite was introduced to test the getvxattr op for newer clients. There
+was a bug where the MDS would crash when a newer client would send a
```

Comment:
A bug existed in the MDS which resulted in the MDS crashing when a newer client sends a getvxattr op (which the MDS did not understand).

Reply:
done